index.js

```diff
@@ -17,6 +19,21 @@ module.exports = function (str, cb) {
 	});
 };
 
+module.exports.stream = function () {
+	var size = 0;
+	return zlib.createGzip(opts)
+		.on('data', function (buf) {
+			size += buf.length;
+		})
+		.on('error', function () {
+			this.size = 0;
+		})
+		.on('end', function () {
+			this.size = size;
+			this.emit('size', size);
+		});
+};
```

Comment:
Ava isn't discontinued, just never published, and I haven't had time to open source it yet. Using tap is fine.